<?php defined('BASEPATH') OR exit('No direct script access allowed'); ?>

ERROR - 2021-04-14 21:24:02 --> Severity: error --> Exception: Call to undefined function mysqli_init() E:\www\joywayid\system\database\drivers\mysqli\mysqli_driver.php 126
ERROR - 2021-04-14 21:24:02 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of Error given, called in E:\www\joywayid\system\core\Common.php on line 658 and defined in E:\www\joywayid\system\core\Exceptions.php:190
Stack trace:
#0 E:\www\joywayid\system\core\Common.php(658): CI_Exceptions->show_exception(Object(Error))
#1 [internal function]: _exception_handler(Object(Error))
#2 {main}
  thrown E:\www\joywayid\system\core\Exceptions.php 190
ERROR - 2021-04-14 21:24:38 --> Severity: error --> Exception: Call to undefined function mysqli_init() E:\www\joywayid\system\database\drivers\mysqli\mysqli_driver.php 126
ERROR - 2021-04-14 21:24:38 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of Error given, called in E:\www\joywayid\system\core\Common.php on line 658 and defined in E:\www\joywayid\system\core\Exceptions.php:190
Stack trace:
#0 E:\www\joywayid\system\core\Common.php(658): CI_Exceptions->show_exception(Object(Error))
#1 [internal function]: _exception_handler(Object(Error))
#2 {main}
  thrown E:\www\joywayid\system\core\Exceptions.php 190
ERROR - 2021-04-14 21:24:49 --> Severity: error --> Exception: Call to undefined function mysqli_init() E:\www\joywayid\system\database\drivers\mysqli\mysqli_driver.php 126
ERROR - 2021-04-14 21:24:49 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of Error given, called in E:\www\joywayid\system\core\Common.php on line 658 and defined in E:\www\joywayid\system\core\Exceptions.php:190
Stack trace:
#0 E:\www\joywayid\system\core\Common.php(658): CI_Exceptions->show_exception(Object(Error))
#1 [internal function]: _exception_handler(Object(Error))
#2 {main}
  thrown E:\www\joywayid\system\core\Exceptions.php 190
ERROR - 2021-04-14 21:26:45 --> Severity: error --> Exception: Call to undefined function mysqli_init() E:\www\joywayid\system\database\drivers\mysqli\mysqli_driver.php 126
ERROR - 2021-04-14 21:26:45 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of Error given, called in E:\www\joywayid\system\core\Common.php on line 658 and defined in E:\www\joywayid\system\core\Exceptions.php:190
Stack trace:
#0 E:\www\joywayid\system\core\Common.php(658): CI_Exceptions->show_exception(Object(Error))
#1 [internal function]: _exception_handler(Object(Error))
#2 {main}
  thrown E:\www\joywayid\system\core\Exceptions.php 190
ERROR - 2021-04-14 21:26:46 --> Severity: error --> Exception: Call to undefined function mysqli_init() E:\www\joywayid\system\database\drivers\mysqli\mysqli_driver.php 126
ERROR - 2021-04-14 21:26:46 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of Error given, called in E:\www\joywayid\system\core\Common.php on line 658 and defined in E:\www\joywayid\system\core\Exceptions.php:190
Stack trace:
#0 E:\www\joywayid\system\core\Common.php(658): CI_Exceptions->show_exception(Object(Error))
#1 [internal function]: _exception_handler(Object(Error))
#2 {main}
  thrown E:\www\joywayid\system\core\Exceptions.php 190
ERROR - 2021-04-14 21:26:48 --> Severity: error --> Exception: Call to undefined function mysqli_init() E:\www\joywayid\system\database\drivers\mysqli\mysqli_driver.php 126
ERROR - 2021-04-14 21:26:48 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of Error given, called in E:\www\joywayid\system\core\Common.php on line 658 and defined in E:\www\joywayid\system\core\Exceptions.php:190
Stack trace:
#0 E:\www\joywayid\system\core\Common.php(658): CI_Exceptions->show_exception(Object(Error))
#1 [internal function]: _exception_handler(Object(Error))
#2 {main}
  thrown E:\www\joywayid\system\core\Exceptions.php 190
ERROR - 2021-04-14 21:26:48 --> Severity: error --> Exception: Call to undefined function mysqli_init() E:\www\joywayid\system\database\drivers\mysqli\mysqli_driver.php 126
ERROR - 2021-04-14 21:26:48 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of Error given, called in E:\www\joywayid\system\core\Common.php on line 658 and defined in E:\www\joywayid\system\core\Exceptions.php:190
Stack trace:
#0 E:\www\joywayid\system\core\Common.php(658): CI_Exceptions->show_exception(Object(Error))
#1 [internal function]: _exception_handler(Object(Error))
#2 {main}
  thrown E:\www\joywayid\system\core\Exceptions.php 190
ERROR - 2021-04-14 21:26:49 --> Severity: error --> Exception: Call to undefined function mysqli_init() E:\www\joywayid\system\database\drivers\mysqli\mysqli_driver.php 126
ERROR - 2021-04-14 21:26:49 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of Error given, called in E:\www\joywayid\system\core\Common.php on line 658 and defined in E:\www\joywayid\system\core\Exceptions.php:190
Stack trace:
#0 E:\www\joywayid\system\core\Common.php(658): CI_Exceptions->show_exception(Object(Error))
#1 [internal function]: _exception_handler(Object(Error))
#2 {main}
  thrown E:\www\joywayid\system\core\Exceptions.php 190
ERROR - 2021-04-14 21:26:50 --> Severity: error --> Exception: Call to undefined function mysqli_init() E:\www\joywayid\system\database\drivers\mysqli\mysqli_driver.php 126
ERROR - 2021-04-14 21:26:50 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of Error given, called in E:\www\joywayid\system\core\Common.php on line 658 and defined in E:\www\joywayid\system\core\Exceptions.php:190
Stack trace:
#0 E:\www\joywayid\system\core\Common.php(658): CI_Exceptions->show_exception(Object(Error))
#1 [internal function]: _exception_handler(Object(Error))
#2 {main}
  thrown E:\www\joywayid\system\core\Exceptions.php 190
ERROR - 2021-04-14 21:26:52 --> Severity: error --> Exception: Call to undefined function mysqli_init() E:\www\joywayid\system\database\drivers\mysqli\mysqli_driver.php 126
ERROR - 2021-04-14 21:26:52 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of Error given, called in E:\www\joywayid\system\core\Common.php on line 658 and defined in E:\www\joywayid\system\core\Exceptions.php:190
Stack trace:
#0 E:\www\joywayid\system\core\Common.php(658): CI_Exceptions->show_exception(Object(Error))
#1 [internal function]: _exception_handler(Object(Error))
#2 {main}
  thrown E:\www\joywayid\system\core\Exceptions.php 190
